BootloaderCommonPkg/Library/IasImageLib/IasImage.c

```diff
@@ -37,11 +38,14 @@ IsIasImageValid (
 {
   EFI_STATUS                 Status;
   IAS_HEADER                 *Hdr;
+  IAS_SIGNATURE_INFO         *Sign;
```

Comment:
IasImage.c is used only for legacy IAS format, no plan to support 3K and others. Please double check if we really need update this file.

Reply:
Yes, I understand  IAS format would be deprecated soon.  We are updating IAS to maintain compatibility (for 2K) as long it would be maintained.